src/main/java/com/tac/guns/entity/ProjectileEntity.java

```diff
@@ -37,7 +36,6 @@
 import net.minecraft.entity.EntityType;
 import net.minecraft.entity.LivingEntity;
 import net.minecraft.entity.Pose;
-import net.minecraft.entity.item.ArmorStandEntity;
```

Comment:
is this cz of IntelliJ's "auto-optimize imports"?

Reply:
I would think, if this is projectile entity this must be from where I started to test more complex bullet physics (Will be creating a separate branch for shortly), this is still an auto import and will be cleared before final push.